README.md

```diff
@@ -16,7 +16,9 @@ Below is a sample `settings.json` file to be included in the root of your Meteor
 			"debug_data": true,
 			"ddp_url": "http://drupalddp.dev",
 			"restws_user": "restws_xxxxx",
-			"restws_pass": "your_password"
+			"restws_pass": "your_password",
+			"simple_security": true,
+			"simple_security_token": "your_security_token"

```

Comment:
@shrop I've updated the simple security token with a sample token. Description about using simple security was added as well.